app/scripts/components/common/map/controls/aoi/custom-aoi-control.tsx

```diff
@@ -168,17 +185,18 @@ function CustomAoI({
     });
     const pids = mbDraw.add(fc);
     setPresetIds(pids);
-    mbDraw.changeMode('simple_static', {
+    mbDraw.changeMode('static_mode', {
```

Comment:
⛏️ : Since you created the constant literal string `STATIC_MODE` do you want to replace these hard coded places with that?